src/core/Feature.js

```diff
@@ -39,9 +41,10 @@ module.exports = class Feature {
     this.default = opts.default;
     this.description = opts.description;
 
-    this.subsection = opts.subsection ? opts.subsection : "general";
+    this.subsection = opts.subsection;
```

Comment:
What will happen if feature files don't provide a subsection, it won't compile?

Reply:
This property is not used anywhere so nothing will happen. It would only make sense to ever be used in the popup, in which case it would be checked if set similarly to the configurables property.